inspire/base/searchext/mappings/hep.json

```diff
@@ -121,7 +121,8 @@
                     "type": "string"
                 },
                 "authors": {
-                    "type": "object",
+                    "type": "nested",
+                    "include_in_parent": true,
```

Comment:
Yeah, as far as I understood `"include_in_parent": true` keeps the intended behavior for what we have done until now. Nested type is good if we need to query array items separately we haven't needed it yet but it could be useful in the future.